cluster_kwargs.yaml

```diff
@@ -31,6 +31,12 @@ parquet_cluster:
   n_workers: 15
   worker_vm_types: [m5.xlarge]  # 4 CPU, 16 GiB
 
+# For tests/workflows/test_embarrassingly_parallel.py
+embarrassingly_parallel_cluster:
+  n_workers: 100
```

Comment:
I am slightly concerned about cost and CI runtime. 
Looking at the cluster, this uses up about 50-100 credits and I believe this one job runs for 10-20min.
I don't think this should run on every commit.

Reply:
We should probably separate the real-world benchmarks from the artificial ones, and then only run the real-world ones (probably they're larger in general) on-demand ?

Reply:
it should be easy to do, this is all pytest stuff so we can use markers, paths, etc.

I'm fine with running on-demand and maybe once per week or release

Reply:
For the record, the cost is about $1-3 bucks per run when using the t3 instances the times are in the order of 5-10 min. I suggested using m6i instances which get a more consistent time usage ~4min every time, but James mentioned he wanted to use the default t3s. For some info on cost and times, you can see this comment from Sarah who run this workflow multiple times for the arm vs intel comparison. 
https://github.com/coiled/platform/issues/645#issuecomment-1459019060

Reply:
I've updated things so that workflows (i.e. all tests in the `tests/workflows/` directory) are only run when requested and on the nightly cron job. Locally you can request workflows be run by adding the `--run-workflows` flag to your `pytest ...` command, and on PRs you can request workflows be run by adding the `workflows` label